src/call.c

```diff
@@ -1194,6 +1194,45 @@ int call_modify(struct call *call)
 	return err;
 }
 
+void call_redirect(struct call *call, uint16_t scode, const char *reason,
+		   const char *contact_params)
```

Comment:
is this function needed ?

can you call `call_hangup` directly ?

Reply:
> is this function needed ?
> 
> can you call `call_hangup` directly ?

I don't think so, because Contact header need to be added and `call_hangup()` passes NULL as extra SIP headers param.